routing.rst

```diff
@@ -20,12 +20,6 @@ the change is simple.
 Creating Routes
 ---------------
 
-First, add support for annotations via the SensioFrameworkExtraBundle:
```

Comment:
This is still a requirement unless one need only routing annotation and should then depend on doctrine annotation or doctrine common as this bundle does.
We should adapt the note but we cannot remove it.

Reply:
This page is about routing. So why exactly is this still a requirement for routing?

Reply:
Because the routing component or the basic skeleton don't rely on doctrine annotations by default.

Reply:
I've reworded as requested by @HeahDude. Also, we don't use Flex shortcuts in the docs to make everything explicit, but we could make an exception in this case with `composer require annotations`

Reply:
I'm 👍 for the exception here. Thanks!

Reply:
Why don't we require `doctrine/annotations` here instead?